nmaipy/parcels.py

```diff
@@ -116,13 +116,21 @@
     },
 }
 
-TREE_BUFFERS_M = dict(
+BUFFER_ZONES_M = dict(
+    buffer_0ft=0.0,
     buffer_5ft=1.524,
     buffer_10ft=3.048,
     buffer_30ft=9.144,
     buffer_100ft=30.48,
 )
 
+BUFFER_CLASSES = {
+    "tree": VEG_MEDHIGH_ID,
+    "woody_veg": VEG_WOODY_COMPOSITE_ID,
```

Comment:
@mbewley where can I validate that this class ID is the one we want for defensible space? The composite layer that was selected during validation. I see in `contants.py` that it is `VEG_WOODY_COMPOSITE_ID = "30fc0c55-2b61-569f-b424-44082987ecb9"` but how do I check what that is?  cc @sjabarnia

Reply:
Nevermind - @sjabarnia confirmed that this is correct ✅

Reply:
@mlopeman If you take a look at the system release report, that class ID is mentioned as the composite class used for fire risk: https://github.com/nearmap/ai_systems_releases/blob/dfa09e465175b06046160ad5ba4b26a31e537cae/products/Gen6/gen6-glowing_grove/gen6-glowing_grove.md

Reply:
Ah, looks like you commented right before I hit submit! Ignore

Reply:
You can also check the api, which is the source of truth, and is used to generate the system release report (there's an nmaipy command that wraps it to get the data frame of classes)

Reply:
Sounds good. For future reference - where would I go to check the API for this?